backend/handler/metadata/moby_handler.py

```diff
@@ -81,38 +81,44 @@ def __init__(self) -> None:
         self.platform_url = "https://api.mobygames.com/v1/platforms"
         self.games_url = "https://api.mobygames.com/v1/games"
 
-    def _request(self, url: str, timeout: int = 120) -> dict:
+    async def _request(
+        self, requests_client: httpx.AsyncClient, url: str, timeout: int = 120
```

Comment:
there's no cleaner way to do this, like initializing the request client in a decorator? or is there a perf loss from initializing each time a function is called?

Reply:
Good call! I have migrated the implementation to use a `ContextVar` where the `httpx` async client is persisted to be reused between requests.

This approach will be useful to share other services in the future, without requiring passing these objects around.